docs/content/docs/(root)/guides/model-context-protocol.mdx

```diff
@@ -1,20 +1,27 @@
 ---
-title: "Connect Your React App to MCP Servers"
+title: "Connect to MCP Servers"
 description: "Integrate Model Context Protocol (MCP) servers with your existing React application for enhanced AI capabilities"
 icon: "lucide/Network"
 ---
 
+<Callout type="info" title="Are you new to CopilotKit?">
+  check out our [Quickstart Guide](https://docs.copilotkit.ai/quickstart) for
+  more details.
+</Callout>
+
 ## What is Model Context Protocol (MCP)?
 
 Model Context Protocol (MCP) is a standard that allows AI models to discover and use tools from external servers. By integrating MCP support, CopilotKit can connect to these servers and leverage their capabilities, expanding what your AI assistant can do.
 
 ## Setup MCP with CopilotKit
 
-Setting up MCP with CopilotKit requires both frontend and backend configuration.
+To set up MCP with CopilotKit, add the `mcpEndpoints` property to your `CopilotKit` component:
 
-### Frontend Setup
-
-Add the `mcpEndpoints` property to your `CopilotKit` component:
+<Callout type="info" title="Get a Public API Key">
```

Comment:
Is this required? My understanding is no, a user doesn't have to use cloud so we should make that clear but default to using cloud.